ui/src/dialogs/DocumentUploadDialog/DocumentUploadForm.jsx

```diff
@@ -86,6 +86,19 @@ export class DocumentUploadForm extends PureComponent {
             }}
           />
         </p>
+        <p className="DocumentUploadForm__info">
```

Comment:
Is the class name actually used anywhere? Think it might be leftover from when you had custom styles applied to this element?

Reply:
I wasn't using it, I thought this was the way to do it, perhaps for accesibility, but I see how that doesn't make any sense now. Removing it :)